broadcast/src/linked/signer/metrics.rs

```diff
@@ -0,0 +1,83 @@
+use commonware_utils::{hex, metrics::status, Array};
+use prometheus_client::{
+    encoding::EncodeLabelSet,
+    metrics::{counter::Counter, family::Family, gauge::Gauge},
+};
+
+/// Label for sequencer height metrics
+#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
+pub struct SequencerLabel {
+    /// Hex representation of the sequencer's public key
+    pub sequencer: String,
+}
+
+impl SequencerLabel {
+    /// Create a new sequencer label from a public key
+    pub fn from<A: Array>(sequencer: &A) -> Self {
+        Self {
+            sequencer: hex(sequencer),
```

Comment:
I think you should call `sequencer.to_string()` here (to leave it up to the `Array` on how it wants to be represented by implementing `Display`).